desafio1.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html>
+<!----------------------------------MENU TOPO--------------------------------------------->    
+    <head>
+        <meta charset="utf-8">
+        <header>
```

Comment:
@tamiresguesdri, use a tag `<header>` dentro da tag `<body>`. O `<head>` usamos para definir as meta tags e  importação de arquivos CSS, incluir o título da nossa página. 

```suggestion
    <body> 
        <header>
            <figure>
                <img src="" alt="ImagemBorboletaWomakersCode"/>
            </figure>
            <nav class="menu-topo">
                <ul>
                    <li> Home </li>
                    <li> Serviços </li>
                    <li> Portfólio </li>
                    <li> Sobre mim </li>
                    <li> Contato </li>
                </ul>
            </nav>
       </header>
   </body>
```